lib/bolt/node/ssh.rb

```diff
@@ -81,6 +81,8 @@ def execute(command, sudoable: false, **options)
             if use_sudo && data == sudo_prompt
               channel.send_data "#{@sudo_password}\n"
               channel.wait
+            elsif use_sudo && data == "Sorry, try again.\n"+sudo_prompt
+              raise "could not execute command: sudo-password not recognized"
```

Comment:
This error handling is not good, but @adreyer is current rewriting how it works so blocking on that work.

Reply:
We can probably add a new kind of error for escalation failures here https://github.com/puppetlabs/bolt/blob/master/lib/bolt/node/errors.rb